src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Windows.cs

```diff
@@ -714,5 +714,29 @@ internal bool IsReplacementCulture
                 return false;
             }
         }
+
+        internal static unsafe CultureData GetCurrentRegionData()
+        {
+            const int BUFF_LENGTH = 10;
+            char* geoIso2Letters = stackalloc char[BUFF_LENGTH];
```

Comment:
It would be better to use `Span<char>` and replace the uses of BUFF_LENGTH with .Length on the Span